app/controllers/tool_cart_controller.rb

```diff
@@ -62,6 +62,18 @@ def checkout
                     locals: {message: "Invalid organization"}
     end
 
+    participant_certs = participant.certifications.map { |cert| cert.certification_type.name }
```

Comment:
This should probably be done when the tool is put into the cart rather than at checkout